src/tests/machine-bind.lua

```diff
@@ -545,7 +541,7 @@ end)
 print("\n\n dump pmas to files")
 do_test("there should exist dumped files of expected size", function(machine)
     -- Dump pmas to files
-    machine:dump_pmas()
+    machine:dump_memory_ranges()
 
     for i = 1, #pmas_file_names do
```

Comment:
This hardcoded list of pmas could be moved inside this test. Morever the `pmas_sizes` could be extracted from `pmas_file_names`. Or the pmas could be listed from `get_memory_ranges`.